libs/graph_utils/src/graph_utils.rs

```diff
@@ -70,10 +70,10 @@ pub fn generate_trace_graph_from_headers(
     properties_header: String,
 ) -> Graph<(String, HashMap<String, String>), String> {
     let mut graph = Graph::new();
-    let mut nodes_iterator = paths_header.split_whitespace();
+    let mut nodes_iterator = paths_header.split(",");
     let mut node_str_to_node_handle = HashMap::new();
     let first_node = nodes_iterator.next();
-    if first_node.is_none() {
+    if first_node.is_none() || first_node.unwrap().trim().is_empty() {
```

Comment:
Why is this needed?

Reply:
The first plugin to receive an RPC will get an empty paths header because the RPC hasn't visited anything yet.  This is checking if it is empty, because if so, we can just return.

Reply:
Why is there a header if it is empty? Should it exist at that point?